### ISSUE TITLE
Filename with special characters or emojis are ommitted when uploaded from mobile client and don't show up in web and mobile client

### ISSUE THREAD

**Issue by Prateek93a** [RocketChat/Rocket.Chat#17245]
Hi, this is related to issue https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/1952 . 

I have raised the PR https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/2020 for solving the issue on mobile client.

 However that causes the filename to appear in encoded form on web client
![Screenshot from 2020-04-10 14-13-58](https://user-images.githubusercontent.com/44807945/78977389-afbb6b00-7b35-11ea-90ab-f13231efd65f.png)

**Comment by lelvisl** [comment]
I don't think this is correct - related to issue https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/2160
